Library/Formula/slimerjs.rb

```diff
@@ -25,25 +49,34 @@ def install
     bin.install_symlink libexec/"slimerjs"
   end
 
-  def caveats; <<-EOS.undent
-    You can set the environment variable SLIMERJSLAUNCHER to a installation of
-    Mozilla Firefox (or Mozilla XULRunner) to use this version with SlimerJS
-    instead of the one installed by Homebrew (this is required if built without
-    xulrunner).
-    For a standard Mozilla Firefox installation this would be:
+  def caveats
+    s = ""
 
-      export SLIMERJSLAUNCHER=/Applications/Firefox.app/Contents/MacOS/firefox
+    ffpath = firefox_installation
```

Comment:
I'm just getting

```ruby
Error: undefined local variable or method `firefox_installation' for #<Formulary::Formulae::FirefoxDependency: "firefox" []>
```

Which is probably something stupidly simple on my part that I haven't yet spotted but will probably want to hit myself in the face with a frying pan once I do spot.